### DISCUSSION TITLE
Dịch Extra-Trees

### DISCUSSION THREAD

**Discussion by cuongvng** [mlbvn/handson-ml2-vn#75] [Terminology] [answered]
Trong chương 7 đang để là "Cây-Extra" vừa Anh vừa Việt.
Mình đề xuất là **"Cây-Bổ sung"**. Mọi người nghĩ sao? @mlbvn/handson-ml2

**Comment by ngcthuong** [comment]
Em cho thêm context đi ?

**Reply by cuongvng** [reply]
đây anh

![image](https://user-images.githubusercontent.com/30251584/114168574-a3413c80-995a-11eb-8541-9ff3c980d7f0.png)

**Reply by cuongvng** [reply]
chị @nguyenhoa93 đang dịch là "Cây Siêu Ngẫu nhiên" https://github.com/mlbvn/handson-ml2-book/pull/188

**Reply by ngcthuong** [reply]
Mình thấy đây là từ viết tắt của "Extremely Randomized Trees" nên có vẻ Cây Siêu Ngẫu nhiên đúng nghĩa gốc hơn.

**Comment by cuongvng** [comment, marked as answer]
vậy chốt là "Cây Siêu Ngẫu nhiên" nhé mn.
